cis_v120/cis.sp

```diff
@@ -1,16 +1,14 @@
 locals {
-  cis_v120_common_tags = {
+  cis_v120_common_tags = merge (local.gcp_compliance_common_tags, {
```

Comment:
```suggestion
  cis_v120_common_tags = merge(local.gcp_compliance_common_tags, {
```

Reply:
Please be careful of spacing (copy and paste is helpful when moving through similar benchmarks across mods)